packages/blockchain-wallet-v4-frontend/src/layouts/Auth/index.tsx

```diff
@@ -78,15 +79,18 @@ const AuthLayoutContainer = ({
   platform,
   unified
 }: Props) => {
-  // lazy load google captcha
-  useDefer3rdPartyScript(
-    `https://www.google.com/recaptcha/enterprise.js?render=${window.CAPTCHA_KEY}`,
-    {
-      attributes: {
-        nonce: window.nonce
+  if (!ChromePlugin.isPlugin()) {
+    // lazy load google captcha
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useDefer3rdPartyScript(
+      `https://www.google.com/recaptcha/enterprise.js?render=${window.CAPTCHA_KEY}`,
+      {
+        attributes: {
+          nonce: window.nonce
+        }
```

Comment:
yeah, I know about that pattern/rule. I just can't come up with better solution. Doing the same from inside the hook will cause the same warning because there are other hooks. Any ideas?

Reply:
Maybe put the check in `useDefer3rdPartyScript` hook?

Reply:
done